lib/service.js

```diff
@@ -1,11 +1,13 @@
 'use strict';
 
-const ServiceValidator = require('./validators/service.validator');
+const {
+  isEmpty,
+  isObject,
+} = require('lodash');
```

Comment:
wait ... we don't have `lodash` as a dependency.  I don't believe these can be used here.  For now, I think we're wanting to get by without using lodash ... trying to keep dependencies to a minimum.  We may need to later, but not for now.